program/idl.json

```diff
@@ -1321,6 +1431,47 @@
           }
         ]
       },
+      {
+        "kind": "instructionNode",
+        "name": "withdrawExcessLamports",
+        "docs": ["Withdraws excess lamports from the account."],
+        "accounts": [
+          {
+            "kind": "instructionAccountNode",
+            "name": "sourceAccount",
+            "isWritable": true,
+            "isSigner": false,
+            "docs": ["Account holding excess lamports."]
+          },
+          {
+            "kind": "instructionAccountNode",
+            "name": "destinationAccount",
+            "isWritable": true,
+            "isSigner": false,
+            "docs": ["Destination account for withdrawn lamports."]
+          }
+        ],
+        "arguments": [
+          {
+            "kind": "instructionArgumentNode",
+            "name": "discriminator",
+            "type": { "kind": "numberTypeNode", "format": "u8", "endian": "le" },
+            "defaultValue": { "kind": "numberValueNode", "number": 27 }
```

Comment:
Are you sure this discriminator is correct? It seems there are already some instructions that have this discriminator so it might just be a copy/paste issue.

Also could you make sure that the instruction is added in the right position in the array? For instance, if this is an instruction of discriminator 42, it should be placed after the one with discriminator 41. It's not mandatory for the code to work but it makes instructions easier to find in the future.